contracts/BaseStrategy.sol

```diff
@@ -391,172 +327,51 @@ abstract contract BaseStrategy {
-     *  entire position based on current on-chain conditions.
-     * @dev
-     *  Care must be taken in using this function, since it relies on external
-     *  systems, which could be manipulated by the attacker to give an inflated
-     *  (or reduced) value produced by this function, based on current on-chain
-     *  conditions (e.g. this function is possible to influence through
-     *  flashloan attacks, oracle manipulations, or other DeFi attack
-     *  mechanisms).
-     *
-     *  It is up to governance to use this function to correctly order this
-     *  Strategy relative to its peers in the withdrawal queue to minimize
-     *  losses for the Vault based on sudden withdrawals. This value should be
-     *  higher than the total debt of the Strategy and higher than its expected
-     *  value to be "safe".
-     * @return The estimated total assets in this Strategy.
-     */
     function estimatedTotalAssets() public virtual view returns (uint256);
 
-    /*
-     * @notice
-     *  Provide an indication of whether this strategy is currently "active"
-     *  in that it is managing an active position, or will manage a position in
-     *  the future. This should correlate to `harvest()` activity, so that Harvest
-     *  events can be tracked externally by indexing agents.
-     * @return True if the strategy is actively managing a position.
-     */
     function isActive() public view returns (bool) {
         return vault.strategies(address(this)).debtRatio > 0 || estimatedTotalAssets() > 0;
     }
 
-    /**
-     * Perform any Strategy unwinding or other calls necessary to capture the
-     * "free return" this Strategy has generated since the last time its core
-     * position(s) were adjusted. Examples include unwrapping extra rewards.
-     * This call is only used during "normal operation" of a Strategy, and
-     * should be optimized to minimize losses as much as possible.
-     *
-     * This method returns any realized profits and/or realized losses
-     * incurred, and should return the total amounts of profits/losses/debt
-     * payments (in `want` tokens) for the Vault's accounting (e.g.
-     * `want.balanceOf(this) >= _debtPayment + _profit`).
-     *
-     * `_debtOutstanding` will be 0 if the Strategy is not past the configured
-     * debt limit, otherwise its value will be how far past the debt limit
-     * the Strategy is. The Strategy's debt limit is configured in the Vault.
-     *
-     * NOTE: `_debtPayment` should be less than or equal to `_debtOutstanding`.
-     *       It is okay for it to be less than `_debtOutstanding`, as that
-     *       should only used as a guide for how much is left to pay back.
-     *       Payments should be made to minimize loss from slippage, debt,
-     *       withdrawal fees, etc.
-     *
-     * See `vault.debtOutstanding()`.
-     */
+    function healtyHarvest() public virtual view returns (bool) {
+        return true;
```

Comment:
can you add more description around this, is the expectation that extending contract should override this logic?

Should it have override explicitly?

Reply:
I added some doc but everything got removed, I have to see what's going on with the linter.
We can have two options for this:

```
    function healtyHarvest() public virtual view returns (bool) {
        return true;
    }
```

OR
```
    function healtyHarvest() public virtual view returns (bool)
```

If we go for the second one without the implementation that will force every strategy to implement the `healtyHarvest` function.

@Grandthrax @retro-wavey